Applications/Games/STAR WARS- Empire at War - Gold Pack/application.json

```diff
@@ -1,4 +1,4 @@
 {
-	"name":	"STAR WARS™ Empire at War - Gold Pack",
+	"name":	"STAR WARS- Empire at War - Gold Pack",
```

Comment:
You should leave it like it has been before.
Also I just noticed that the folder is called "STAR WARS- Empire at War - Gold Pack". Looks a bit strange without space before "-". Could you change it to "STAR WARS Empire at War - Gold Pack" ?